ui/src/views/AwsProvisioner/ViewAwsWorkerType/index.jsx

```diff
@@ -142,12 +149,15 @@ export default class ViewAwsWorkerType extends Component {
           <AwsProvisionerErrorsTable errors={awsProvisionerWorkerTypeErrors} />
         )}
         {!error && !loading && currentTab === 2 && (
-          <Ec2ResourcesTable
-            onTerminateInstance={this.handleTerminateInstance}
-            workerType={awsProvisionerWorkerType}
-            awsState={awsProvisionerWorkerTypeState}
-            actionLoading={actionLoading}
-          />
+          <FixedSizeList>
+            <Spinner className={classes.spinner} {...tabContentLoader} />
```

Comment:
`FixedSizeList` should take a function as a child. See https://github.com/taskcluster/taskcluster/blob/7b058a76c4a92305c85083cfbcf19effcb53603a/ui/src/components/RoleScopesTable/index.jsx#L142 for an example. You can find examples and prop definitions in https://react-window.now.sh/#/api/FixedSizeList.

Reply:
I also think the `FixedSizeList` should be in the render method of `Ec2ResourcesTable`.

Reply:
I will do the needful!